packages/engine/src/datastore/batch/migration/mod.rs

```diff
@@ -1418,381 +1418,350 @@ pub(super) mod test {
         }
     }
 
-    #[cfg(test)]
-    pub(super) mod test {
```

Comment:
The git diff is big because of this, no code changes were made outside of removing the nesting and commented imports

Reply:
You can hide whitespace changes:
![image](https://user-images.githubusercontent.com/21277928/152837949-6b4e5aa0-391c-4b34-8b1c-df69ccdced54.png)